core/block_rendering_rewrite/block_render_info.js

```diff
@@ -316,13 +316,16 @@ Blockly.BlockRendering.RenderInfo.prototype.getInRowSpacing_ = function(prev, ne
     if (prev.isField() && prev.isEditable) {
       return BRC.MEDIUM_PADDING;
     }
+    // A block with just an icon is skinny.  Make it more substantial.
```

Comment:
What if it's two or three icons? Is this adding necessary padding after the last icon or just bumping up the minimum block length? If the latter, should the block minimum width be bigger even for empty blocks?

Reply:
This is adding padding after the last icon no matter what.  I think that's the right thing to do, because then a block with lots of icons still noticeably looks like a block instead of a blob.

Here's (some of) the code that controls this from the old rendering:
```
  inputRows.rightEdge = iconWidth + Blockly.BlockSvg.SEP_SPACE_X * 2;
  if (this.previousConnection || this.nextConnection) {
    inputRows.rightEdge = Math.max(inputRows.rightEdge,
        Blockly.BlockSvg.NOTCH_WIDTH + Blockly.BlockSvg.SEP_SPACE_X);
  }
```

Min width for empty blocks: Empty blocks look like this right now:
![image](https://user-images.githubusercontent.com/13686399/58904034-96e35d00-86bb-11e9-9d75-c423aa3752ea.png)

which matches the old rendering and (imo) has enough width to show that it's a block and not a sliver.

Reply:
sgtm. Just update the comment, then.

Reply:
Done.